### PR TITLE
another (lava)nazi pr

### PR DESCRIPTION
**Pull request by MMMiracles**
A super-massive complex, heavily fortified and guarded by an organization of old-old-Earth fascists. If you think you're going to solo this monstrosity or with a few of your mining pals, man you're so wrong. Highly advised bringing a good part of the crew if you want to get through this fortress of questionable moral activities. 

Coming in at a whopping 90x90 size, this fortress of fascism sports a whole variety of player-inducing salt, with some phat loot for those who somehow manage to get through most of it without dying. Also removes the dinky outpost because it was mostly just placeholder till I could finish this.

It's basically done so at this point I'm just looking for loot-balance advice. My main concern is teleporting into the phat loot so if someone could explain how to prevent TPing into loot-sensitive areas that would be great. 

Total PRs with nazi-related content: 5 (no agenda here sir)

### CODE REVIEW

**Comment by Shadowlight213** [comment]
Inb4 rule -1. "No nazis"

**Comment by Man-Shroom-** [comment]
pic?

**Comment by kevinz000** [comment]
Bring back YBLT and we'Lloyd test the "don't solo this" theory.

**Comment by MMMiracles** [comment]
@Man-Shroom- 
its big and would require me to stitch screenshots together so ugghhhh

download the branch if you really want to ruin the surprise of seeing it in-game first few times unless a maintainer yells at me to post a picture

**Comment by kevinz000** [comment]
Also block telescience bluecrystals and hand telescience and RTA

**Comment by MMMiracles** [comment]
@kevinz000 already addressed this concern in the main comment, only thing I could find of relevance related to blocking teleporting was a variable that wasn't used by anything but crew dorms.

**Comment by Aloraydrel** [comment]
banned from germany

**Comment by KorPhaeron** [comment]
Gotta wait till nullbears PR is merged for the noteleport.

Also no need for spoiler images.

**Comment by Cheridan** [comment]
rule 1 no nazis

**Comment by MMMiracles** [comment]
i mean they're obviously not nazis, they're wwii reenactors.

**Comment by YakumoChen** [comment]
I'd rather not have a structure taking up literally half the mine and preventing most of lavaland from being usable, to be honest.

**Comment by YakumoChen** [comment]
Didn't we just have a PR that specifically REDUCED the number of gigantic vault spawns?

**Comment by MMMiracles** [comment]
the vault away mission is already a possible ruin so

**Comment by Lumipharon** [comment]
Yeah this is pretty... eh, for single z-level mining.

**Comment by MMMiracles** [comment]
@Lumipharon As far as I'm aware lavaland is meant to be a multi z-level in the long run, that and the fact the vault mission is already a thing that can spawn in.

Besides, the cost of it is going to make it a rare thing to appear if it isn't picked really early on by the ruin spawner, so its not like every other round is going to have it (in theory).

**Comment by KorPhaeron** [comment]
Can't we call them the  New Wehrmacht and say they fles after WW2 then give them giant eagles as symbols instead swastikas?

**Comment by MMMiracles** [comment]
i didn't bother actually bringing over any of the banners or flags from the original PR because of the swastika symbols. there isn't actually anything in this map that directly mentions/references nazis other than mecha-hitler.

**Comment by KazeEspada** [comment]
Obviously we need to change it to be a nazi like organization....

What about a some sort of mythical creature... like maybe a hydra or something.

<sub>hail hydra</sub>

**Comment by KorPhaeron** [comment]
@Cheridan 

Is there some form of notnazis you will accept?

**Comment by Lumipharon** [comment]
Just make them EVIL FASCISTS.

**Comment by MMMiracles** [comment]
I like kor's idea of 'New Wehrmachts', just hoping Cheridan is actually willing to explain what is allowed and what isn't.

**Comment by Dorsisdwarf** [comment]
Syndicates are already Hydra have you _seen_ that badass background picture for the uplink?

**Comment by MMMiracles** [comment]
i honestly find this better tbh

**Comment by Lumipharon** [comment]
>Not making the enemies coders
>not making the fortress the coderbus

Please, there's so much potential for rage inducing hilarity there.

**Comment by MMMiracles** [comment]
you see im actually trying to get this merged in the near future

**Comment by KorPhaeron** [comment]
Having a bunch of transhumanists who call themselves Coders and do genetic experiments would be pretty funny.

I still want to shoot nazis in hell though

**Comment by MMMiracles** [comment]
i'll look into expanding the deep lore of lava nazis tomorrow probably, do some touch-ups on the ruin as well before i call it done until the noteleport gets merged, @nullbear pls

**Comment by Cheridan** [comment]
Yes @KorPhaeron Not-Nazis are ok

**Comment by MMMiracles** [comment]
so can we get that silly old 'DNM' label removed or

**Comment by Lumipharon** [comment]
@KorPhaeron And we could reference old pr's that changed things significantly, or were proposed, but never got merged. Like oldrun speed being in effect inside the bus via reskinned bluespace floors, etc.

I unno, I'm just in the mode for something slightly meta and fun, since this is supposedly to be quite rare.

**Comment by KorPhaeron** [comment]
@MMMiracles 

Can we do New Wehrmacht now then VORE Gestapo is a bet over the top for me

File: code/modules/mob/living/simple_animal/hostile/reenactors.dm [hunk 1]
```diff
@@ -5,8 +5,8 @@
 */
 
 /mob/living/simple_animal/hostile/wwii
-	name = "World War II Reenactor"
-	desc = "A VERY enthusiastic WWII reenactor."
+	name = "Vorestation Gestapo"
```
Comment (Cheridan) [inline code/modules/mob/living/simple_animal/hostile/reenactors.dm]: oh my god you really did it

File: code/modules/mob/living/simple_animal/hostile/reenactors.dm [hunk 2]
```diff
@@ -57,24 +57,30 @@
...
-			C.Weaken(3)
-			C.adjustBruteLoss(10)
-			C.visible_message("<span class='danger'>\The [src] smashes \the [C] into the ground!</span>", \
-					"<span class='userdanger'>\The [src] smashes you into the ground!</span>")
-			src.say(pick("RAAAAAGGHHHH!!!","AAAARRRGGHHHH!!!","RRRAAUUUGGHH!!!"))
+		var/mob/living/C = target
+		if(C.stat == DEAD)
+			C.gib()
+			visible_message("<span class='danger'>[src] devours [C] whole!</span>")
+			src << "<span class='userdanger'>You devour [C] whole, restoring your health!</span>"
```
Comment (Cheridan) [inline code/modules/mob/living/simple_animal/hostile/reenactors.dm]: oh my gooooooooooooooddddddddddddddddddd

**Comment by MMMiracles** [comment]
I will never be outdone.

**Comment by Lumipharon** [comment]
oh my.

File: code/modules/mob/living/simple_animal/hostile/reenactors.dm (hunk 2 above)
Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/hostile/reenactors.dm]: wow you stole my dragon code I'm calling github

Comment (MMMiracles) [inline code/modules/mob/living/simple_animal/hostile/reenactors.dm]: no that uses an L for the targeted mob I use a C its completely different sheesh

**Comment by WJohn** [comment]
Please give us a map pic, I'm not able to review this thing at the moment since my internet doesn't work at home and my codebase is super outdated.

You really should get used to making map pics, it's necessary for quality control, I know that you've definitely gotten some less-than-stellar mapping through because of lacking pictures.

**Comment by MMMiracles** [comment]
Kor already said no need for spoiler images, so I didn't bother putting it together. 

If it's absolutely necessary I'll stitch one together later and send you it on coderbus or something.

**Comment by MMMiracles** [comment]
Transhumanist coders is funny and I'll totally look into that after the freeze but for now we have not-nazis.

**Comment by Lumipharon** [comment]
we're on an open code base and it will get posted eventually if people genuinely wan to powergame it or some shit - there's no point in trying to keep anythin secret because it will stay secret for precisely 5 seconds anyway tbh.

**Comment by MMMiracles** [comment]
I mean anyone could also download as if they really want to spoil it now. 

Gonna fix the conflict real quick and get a screen so WJohnson  can't call my mapping sub-par solely for not having a picture (I don't even honestly)

**Comment by Lumipharon** [comment]
Because not everyone can access dream maker at all times, also you can use the argument of 'download as is' for example, to not describe what the pr does what so ever "just check the code :^)"

Not criticizing you, just saying.

**Comment by KorPhaeron** [comment]
@MMMiracles 

Is this done

**Comment by MMMiracles** [comment]
i mean other than the lack of teleport cheese protection, yes

**Comment by KorPhaeron** [comment]
oh right I keep forgetting @nullbear please

**Comment by MMMiracles** [comment]
welp its been open for over 22 days and oranges closed it

reeeeee

**Comment by optimumtact** [comment]
nullbear said they would finish the work

**Comment by nullbear** [comment]
notp is good.

**Comment by Cheridan** [comment]
i changed my mind, rule -1 stands

**Comment by PKPenguin321** [comment]
cheridan more like anti-nazi nazi-dan

**Comment by MMMiracles** [comment]
what kind of sick game are you playing

w h y

**Comment by Dorsisdwarf** [comment]
Rename them "Coders"

**Pull request closed without merging**